### PR TITLE
Calculator was added

### PR DESCRIPTION
**Pull request by boniface193**
calculator styling
html layout
estlint
react installed

### CODE REVIEW

File: public/robots.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```
Comment (DammyShittu) [inline public/robots.txt]: - [ ] Kindly remove this file and these other CRA files in the public directory: `robots.txt`, `logo512.png`, `logo192.png`.

  image

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            <CalBtn incrementNumber={4} bgColor="bg-white btn p-normal" />
+            <CalBtn incrementNumber={5} bgColor="bg-white btn p-normal" />
+            <CalBtn incrementNumber={6} bgColor="bg-white btn p-normal" />
+            <CalBtn characterStr="-" bgColor="bg-orange btn p-normal" />
+          </div>
+          <div>
+            <CalBtn incrementNumber={1} bgColor="bg-white btn p-normal" />
+            <CalBtn incrementNumber={2} bgColor="bg-white btn p-normal" />
+            <CalBtn incrementNumber={3} bgColor="bg-white btn p-normal" />
+            <CalBtn characterStr="÷" bgColor="bg-orange btn p-normal" />
```
Comment (DammyShittu) [inline src/components/Calculator.js]: - [ ] You interchanged the placements of the plus and divide signs. Kindly fix this.

  pl-2

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @boniface193,

## Highlights
✔️ Good design and colors

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by V-Blaze — APPROVED** [review]
Hi @boniface193  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- Used react functional components💯
-  Correct Gitflow👍
-  UI match the provided design 🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**